client/cypress/end-to-end/audit-setup.spec.js

```diff
@@ -0,0 +1,12 @@
+describe('Audit setup', () => {
+  it('Creates an audit', () => {
+    cy.visit('/')
+    cy.loginAuditAdmin('audit-admin-cypress@example.com')
+    cy.contains('Give your new audit a unique name.').type('Audit One')
```

Comment:
you can use testing library commands like `findByRole` cuz I installed @testing-library/cypress: https://testing-library.com/docs/cypress-testing-library/intro